internal/controllers/topology/cluster/conditions_test.go

```diff
@@ -858,15 +829,10 @@ func TestReconcileTopologyReconciledCondition(t *testing.T) {
 								WithReplicas(2).
 								WithVersion("v1.22.0").
 								WithStatus(clusterv1.MachineDeploymentStatus{
```

Comment:
Intentional change to increase the replica fields to 2?

Reply:
Not sure I follow, I'm not changing replicas to 2

Reply:
Based on this diff they were 1 before
![image](https://github.com/user-attachments/assets/635226e0-d3e1-4fdc-bb41-059c93e7e80b)

Reply:
Thanks, I got confused by WithReplicas(2) above comments.
Fixed